CanonicalLtd/candid
# Update the login templates to match the latest design

## Done
Update the login and register template to use the Vanilla view

## QA
- Run `make server`
- Select default provider
- See the login page shows correctly
- I don't think register view is used yet but updated that too

## Screenshot
![Screenshot_2019-06-20 Candid - Login](https://user-images.githubusercontent.com/1413534/59833111-4a478500-933d-11e9-82d9-6eee66419bc0.png)

Fixes https://github.com/canonical-web-and-design/RBAC-squad/issues/295
Fixes https://github.com/canonical-web-and-design/RBAC-squad/issues/294
